include/fmt/color.h

```diff
@@ -215,6 +234,32 @@ struct rgb {
   uint8_t b;
 };
 
+namespace internal {
+
+// color is a struct of either a rgb color or a terminal color.
+struct color_type {
+  FMT_CONSTEXPR color_type() FMT_NOEXCEPT
+    : is_rgb(), value{} {}
+  FMT_CONSTEXPR color_type(color rgb_color) FMT_NOEXCEPT
+    : is_rgb(true), value{} {
```

Comment:
@vitaut Nevermind my previous comment. They are initialized because they have to be for `constexpr` and it seems like VS2013 doesn't implement directly initializing the union in a member initializer list. So I have to leave it like it is :(